src/mono/sample/wasm/browser-threads-minimal/main.js

```diff
@@ -8,7 +8,10 @@ const assemblyName = "Wasm.Browser.Threads.Minimal.Sample.dll";
 
 try {
     const { setModuleImports, getAssemblyExports, runMain } = await dotnet
-        .withEnvironmentVariable("MONO_LOG_LEVEL", "debug")
+        //.withEnvironmentVariable("MONO_LOG_LEVEL", "debug")
+        .withConfig({
+            pthreadPoolSize: 20,
+        })
```

Comment:
Is the managed threadpool eating all the available threads? That doesn't seem right - this sample should only need about 2 threads...

Reply:
Right now the sample needs 6. 
- 2 are thread in the mono thread pool
- StartTimerFromWorker keeps one ticking
- StartAllocatorFromWorker keeps one ticking
- TestCallSetTimeoutOnWorker keeps one ticking
Mono thread pool only returns threads when allocating new one or on finalizer thread.

I'm not sure if this is detaching threads fast enough.